pkgs/by-name/pa/parca/package.nix

```diff
@@ -0,0 +1,82 @@
+{
+  buildGoModule,
+  faketty,
+  fetchFromGitHub,
+  lib,
+  nodejs,
+  pnpm,
+  stdenv,
+}:
+let
+  version = "0.22.0";
+
+  parca-src = fetchFromGitHub {
+    owner = "parca-dev";
+    repo = "parca";
+    rev = "v${version}";
+    hash = "sha256-iuTlKUmugRum0qZRhuw0FR13iE2qrQegTgwpAvgJSXk=";
+  };
+
+  ui = stdenv.mkDerivation rec {
+    inherit version;
+    pname = "parca-ui";
+    src = "${parca-src}/ui";
+    npmDepsHash = "";
+
+    pnpmDeps = pnpm.fetchDeps {
+      inherit pname src version;
+      hash = "sha256-MVNO24Oksy/qRUmEUoWoviQEo6Eimb18ZnDj5Z1vJkY=";
+    };
+
+    nativeBuildInputs = [
+      nodejs
+      pnpm.configHook
+    ];
+
+    buildInputs = [ faketty ];
```

Comment:
Shouldn't this be in nativeBuildInputs ?

Reply:
Fixed! thanks!